### ISSUE TITLE
When %% is commented JupyterSendCell is not working as expected

### ISSUE THREAD

**Issue by ilkerhk** [jupyter-vim/jupyter-vim#47]
A minor issue:  If there is an indented cell separator inside another cell, then running that cell using JupyterSendCell only runs the code until the indented cell separator. This causes errors, see below example. 

Commented cell separators should be ignored. 
`# %% Cell Block
if 1==1:
    print(1)
else: 
    # %% Disabled Cell Block
    print(2)

# %% Next Cell
`

**Comment by wmvanvliet** [comment]
I agree this would be desired behavior. Cell markers should have the same indentation level.

**Comment by tinmarino** [comment]
PR #65 will fix this removing the `.strip` of [each line content before the match](https://github.com/jupyter-vim/jupyter-vim/pull/65/commits/f9c047f51044a9fa7d80c18d8c0e1f47fe28e5ce). Thanks @ilkerhk, well seen.
